### DISCUSSION TITLE
Cleanup AWS Light Stemcells Hosted in Acc. 138312XXXXXX

### DISCUSSION THREAD

**Discussion by nouseforaname** [cloudfoundry/bosh#2412] [General]
Hello everyone,

we are currently hosting very old Stemcells in the Account mentioned. 

The most recent Stemcells in this account are:

![image](https://user-images.githubusercontent.com/34882943/203515121-9cd22e5c-c910-487d-a518-0f34b4e54020.png)

The latest was created `2018/01/09 04:30`. 

We'd like to start a discussion about deprecating the stemcells in this Account.

Do you see any Issue with cleaning them? Any awareness about current usage of that Account (is OSS CI maybe still referencing artifacts in that account somewhere)?

**Comment by nouseforaname** [comment]
That account also contains these buckets:
![image](https://user-images.githubusercontent.com/34882943/203518350-9cd2aaee-3eb1-41ad-8211-05462e0599a7.png)

**Comment by beyhan** [comment]
I wasn't aware of this account. Most probably it is safe to clean it. @ramonskie do you know of any usage in OSS CI  for the mention artefacts above?

**Comment by ramonskie** [comment]
these buckets and resources are not used in the OSS ci
